### ISSUE TITLE
Fov circle

### ISSUE THREAD

**Issue by pwnmy** [danielkrupinski/Osiris#3892]
Where would i add the code for a fov circle

**Comment by 1310s** [comment]
just use https://github.com/notgoodusename/OsirisAndExtra

**Comment by Xsintashi** [comment]
mans pasting here, without any minimal knowledge ab c++.

**Comment by Xtendera** [comment]
> mans pasting here, without any minimal knowledge ab c++.

Exactly, the code for the PR is a little outdated, so a couple of things have gotta be changed, but overall not that hard for a non-skid. I dont think even half of the people here actually know cpp.

**Comment by Xsintashi** [comment]
> > mans pasting here, without any minimal knowledge ab c++.
> 
> Exactly, the code for the PR is a little outdated, so a couple of things have gotta be changed, but overall not that hard for a non-skid. I dont think even half of the people here actually know cpp.

Imma be honest, I also started pasting osiris without any knowledge about c++, but knew how to change/add basic shit

**Comment by Xtendera** [comment]
> > > mans pasting here, without any minimal knowledge ab c++.
> > 
> > 
> > Exactly, the code for the PR is a little outdated, so a couple of things have gotta be changed, but overall not that hard for a non-skid. I dont think even half of the people here actually know cpp.
> 
> Imma be honest, I also started pasting osiris without any knowledge about c++, but knew how to change/add basic crap

TBRH, I started REALLY modding a couple days ago, I had only messed around a bit in the past. I don't know cpp THAT well, but I am a programmer and have used cpp and rust in the past so I think I can hopefully not be titled a "skid".

I have forked this repo in a private fork and I am adding features as needed (most of them original, some harder ones requiring more math like the draw fov one being added from modified PRs).  I am mainly making this for my freind who plays csgo more than I do. I am currently working on an external UI LOL.

<sub><sup>You tell me, am I still a skid?</sup></sub>

**Comment by 1310s** [comment]
> 

if pasting and modifying a source that does not belong to you turns you into a "skid" we would all be skids. 👾👾👾
